src/components/pages/order/OrderPage.js

```diff
@@ -10,24 +10,24 @@ import GlobalContext from "../../../context/GlobalContext";
 export default function OrderPage() {
   const { name } = useParams();
   const [isModeAdmin, setIsModeAdmin] = useState(false);
-  const [panelTabIndex, setPanelTabIndex] = useState(2);
-  const [isPanelReduce, setIsPanelReduce] = useState(false);
+  const [panelTabIndex, setPanelTabIndex] = useState(1);
```

Comment:
```suggestion
  const [panelTabIndex, setPanelTabIndex] = useState("add");
```

As-tu plus explicite qu'un nombre qui est assez vague ? Quand tu nomme quelque chsoe essaie au maximum de t'exploigner de l'implémentation technique et plus vers le "métier" et le "concret". Le test c'est : "même un non technos sans background technique est censé comprendre". Ici, "panelTabIndex" avec une initialisation à 1 c'est un peu trop technique mais "panelTabIndex" avec "add" on visualise mieux tout de suite.